renv.lock

```diff
@@ -1,6 +1,6 @@
 {
   "R": {
-    "Version": "4.4.1",
+    "Version": "4.3.2",
```

Comment:
Hi Reuben! The pull request looks great except for this change. It looks like you need to update your R version!

Reply:
Thanks, let me update my R version.